src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx

```diff
@@ -40,14 +49,46 @@ function UserStats({totals, location, organization}: Props) {
     const apdexKey = `apdex_${threshold}`;
     const formatter = getFieldRenderer(apdexKey, {[apdexKey]: 'number'});
     apdex = formatter(totals, {organization, location});
+
+    const [vitalsPassed, vitalsTotal] = Object.values(WebVital)
+      .filter(vital => WEB_VITAL_DETAILS[vital].includeInSummary)
+      .reduce(
+        ([passed, total], vital) => {
+          const alias = getAggregateAlias(`percentile(${vital}, ${VITAL_PERCENTILE})`);
+          if (totals[alias] !== null && !isNaN(totals[alias])) {
+            total += 1;
+            if (totals[alias] < WEB_VITAL_DETAILS[vital].failureThreshold) {
+              passed += 1;
+            }
+          }
+          return [passed, total];
+        },
+        [0, 0]
+      );
+    if (vitalsTotal > 0) {
+      vitalsPassRate = <StatNumber>{`${vitalsPassed} / ${vitalsTotal}`}</StatNumber>;
+    }
   }
 
+  const webVitalsRoute = generateWebVitalsRoute({orgSlug: organization.slug});
+
   return (
     <Container>
       <div>
         <SectionHeading>{t('Apdex Score')}</SectionHeading>
         <StatNumber>{apdex}</StatNumber>
       </div>
+      <Feature features={['measurements']} organization={organization}>
+        {vitalsPassRate !== null && (
+          <div>
+            <SectionHeading>{t('Web Vitals')}</SectionHeading>
+            <StatNumber>{vitalsPassRate}</StatNumber>
+            <Link to={`${webVitalsRoute}${location.search}`}>
```

Comment:
Couldn't the `webVitalsRoute` take the querystring parameters and return a LocationDescriptorObject so that you don't need to concatenate strings together?